packages/fiori-elements-writer/README.md

```diff
@@ -0,0 +1,76 @@
+      },
```

Comment:
Prefer a smaller edmx in the readme. Preview looks ok. 
But Readme.md should be small for performance of npm afaik. Maybe put in a seperate md file?

Reply:
Agreed, copied from `fiori-freestyle-writer`, so I guess we should update there also?

Reply:
Probably yes !

Reply:
Probably yes !